plutus-tx-plugin/test/Lift/list.plc.golden

```diff
@@ -2,46 +2,11 @@
   [
     [
       [
-        (lam
-          GHC_Types_Nil_i0
-          (lam
-            GHC_Types_Cons_i0
-            (lam
-              match_GHC_Types_Nil_i0
-              [
-                [ (force GHC_Types_Cons_i2) (con integer 1) ]
-                (force GHC_Types_Nil_i3)
-              ]
-            )
-          )
-        )
-        (delay
-          (delay
-            (lam
-              case_GHC_Types_Nil_i0
-              (lam case_GHC_Types_Cons_i0 case_GHC_Types_Nil_i2)
-            )
-          )
-        )
+        (lam i0 (lam i0 (lam i0 [ [ (force i2) (con integer 1) ] (force i3) ])))
```

Comment:
This test output is unusuable now. Can we keep using named indices for this sort of stuff? We're not evaluating it.